HousingRepairsOnlineApi/Startup.cs

```diff
@@ -51,6 +52,15 @@ public void ConfigureServices(IServiceCollection services)
                 return new AppointmentsGateway(httpClient, authenticationIdentifier);
             });
 
+            var notifyApiKey = GetEnvironmentVariable("GOV_NOTIFY_KEY");
+
+            var notifyClient = new NotificationClient(notifyApiKey);

```

Comment:
This instantiation would be better placed within the `GovNotifyGateway` initialisation lambda (line 59) so that it's only instantiated when required.

If a single `NotificationClient` is required to be shared across all `GovNotifyGateway`s, then consider registering `NotificationClient` as a singleton instead.